<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>178</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>178</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>55</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icon/chalk_bundle.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Adobe Garamond Pro Bold</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CHALK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Adobe 고딕 Std B</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Smart Structures&amp;Systems Lab Labeling Tool</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>Enter User Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
